docs/environment-variables.md

```diff
@@ -70,3 +71,48 @@ FLASK_ENVIRONMENT_CONFIG=dev
 SECRET_KEY='some_random_key'
 (...)
 ```
+
+# Database Environment Variables
+
+To run the backend in other than `local` mode i.e `prod` or `develop` you need to export database environment variables.
```

Comment:
maybe you could add examples of services that one can use to have a remote database set, like a local instance of postgres or mysql, or even ElephantSQL (https://www.elephantsql.com/)